app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
```

Comment:
There must be a blank line between different types of imports. 
Imports from same module can be listed separated by comma instead